index.js

```diff
@@ -127,8 +154,9 @@ NodePreGypGithub.prototype.publish = function(options) {
 		if(err) {console.error(err); return;}
 		
 		release	= (function(){ // create a new array containing only those who have a matching version.
+			var tag_name = options.tag_name || this.package_json.version;
 			data = data.filter(function(element, index, array){
-				return element.tag_name === this.package_json.version;
+				return element.tag_name === tag_name;
```

Comment:
>but the line 63 is in createRelease() which is called only later and options.tag_name might not be intialized (if no --tag command option).

You are correct. Good observation. After this PR, I will move the defaults so they get applied at the beginning of publish() function instead of release(). Makes more sense that way.

>Is this your "deleted comment"? If yes - then please ignore my answer.

no this was not the comment I deleted.